pkg/trade/trade.py

```diff
@@ -299,13 +428,36 @@ def _cumulative_returns(
         self,
         returns: list[float],
     ) -> float:
+        """Calculate the cumulative returns of a list of returns.
+
+        Args:
+            returns (list[float]): A list of returns.
+
+        Returns:
+            float: The cumulative returns rounded to 4 decimal places.
+
+        """
         cumulative_returns = np.prod(1 + np.array(returns)) - 1
 
         return round(cumulative_returns, 4)
 
     def _get_risk_free_rate(
         self,
     ) -> float:
+        """Retrieve the risk-free rate from the Alpha Vantage API.
```

Comment:
Do we want docstrings on the private methods?

Reply:
yea good call. Fixed.